src/deploy/osp_deployer/settings/config.py

```diff
@@ -239,6 +239,17 @@ def __init__(self, settings_file):
         else:
             self.enable_instance_ha = False
         self.overcloud_nodes_pwd = deploy_settings['overcloud_nodes_pwd']
+        dellnfv_settings = self.get_settings_section(
+            "Dell NFV Settings")
+        if dellnfv_settings['hpg_enable'].lower() == 'true':
+            self.hpg_enable = True

```

Comment:
Should we have defaults, what happens if there is no Dell NFV Settings in ini?  So if  get_settings_section doesn't find a particular section there are a set of defaults returned?  I've seen this pattern in other openstack projects.  Perhaps this could be a future refactor.

Reply:
Making sure all stanza/options are present is already covered further up in that class